src/vs/workbench/browser/parts/editor/editorTabsControl.ts

```diff
@@ -102,6 +97,7 @@ export abstract class EditorTabsControl extends Themable implements IEditorTabsC
 	};
 
 	private editorActionsToolbar: WorkbenchToolBar | undefined;
+	private editorActionsChangeDisposable: IDisposable | undefined;
```

Comment:
I suggest to use a `MutableDisposable` here. The advantage is that you can `register` it to the `store` of the `EditorTabsControl` to make sure it gets disposed when the control gets disposed. And when you set `disposable.value` it will assign the new disposable and dispose the old.